xwiki-rendering-syntaxes/xwiki-rendering-syntax-annotatedxhtml/src/main/java/org/xwiki/rendering/internal/renderer/xhtml/AnnotatedXHTMLChainingRenderer.java

```diff
@@ -34,6 +36,11 @@
  */
 public class AnnotatedXHTMLChainingRenderer extends XHTMLChainingRenderer
```

Comment:
There is also metadatas in beginDocument.

Reply:
Not sure to understand the comment here. You want me to ignore the metadata processed by begin/endDocument? Or on the contrary to process them here?

Reply:
Actually forget that. Handling begin/endDocument might do more harm than good right now, we'll see how it goes and if we need it later for some use case.